src/hosts/vscode/hostbridge/window/showTextDocument.ts

```diff
@@ -1,9 +1,29 @@
 import * as vscode from "vscode"
 import { ShowTextDocumentRequest, TextEditorInfo } from "@/shared/proto/host/window"
+import { arePathsEqual } from "@/utils/path"
 
 export async function showTextDocument(request: ShowTextDocumentRequest): Promise<TextEditorInfo> {
 	// Convert file path to URI
 	const uri = vscode.Uri.file(request.path)
+
+	// Check if the document is already open in a tab group that's not in the active editor's column.
+	//  If it is, then close it (if not dirty) so that we don't duplicate tabs
```

Comment:
is it possible to change focus (i mean, move to that tab), not reopen it? i sometimes have meaningful order of tabs, and would be very upset if ide reshuffle them